<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SongPropertiesDialog_UI</class>
 <widget class="QDialog" name="SongPropertiesDialog_UI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>290</width>
    <height>378</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form1</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="TextLabel1">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
     <property name="text">
      <string>Song name</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLineEdit" name="songNameTxt"/>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLabel" name="TextLabel1_2">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
     <property name="text">
      <string>Author</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QLineEdit" name="authorTxt"/>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QLabel" name="TextLabel1_3">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
     <property name="text">
      <string>License</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QLineEdit" name="licenseTxt"/>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QLabel" name="TextLabel2">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
     <property name="text">
      <string>Notes</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QTextEdit" name="notesTxt"/>
   </item>
   <item row="8" column="0">
    <widget class="QPushButton" name="cancelBtn">
     <property name="text">
      <string>&amp;Cancel</string>
     </property>
     <property name="shortcut">
      <string>Alt+C</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QPushButton" name="okBtn">
     <property name="text">
      <string> &amp;OK</string>
     </property>
     <property name="shortcut">
      <string>Alt+O</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
